src/figuresClasses.ts

```diff
@@ -1,11 +1,78 @@
-export interface Figure {}
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
+export class Triangle implements Figure {
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+    public shape: Shape = 'triangle',
```

Comment:
The 'shape' property is being set to a default value of 'triangle' which is redundant since the Triangle class should always represent a triangle. You can remove this default value and declare the 'shape' property in the class body with a readonly modifier and a constant value.